packages/components/organisms/f-footer/CHANGELOG.md

```diff
@@ -3,534 +3,557 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-v4.6.0
-------------------------------
-*January 12, 2021*
```

Comment:
Could you please return back the format to stay the same with all the other components we have.

Reply:
Hum, not sure what happened here. I may have some linting, or file formatting rule on Markdown. 

Will do, thank you.